package.json

```diff
@@ -79,6 +79,7 @@
     "babel-polyfill": "^6.26.0",
     "ethereumjs-util": "6.0.0",
     "jsbi-utils": "^1.0.0",
+    "lodash": "^4.17.11",
```

Comment:
To reduce bloat I suggest to use these:
https://www.npmjs.com/package/lodash.isfunction
https://www.npmjs.com/package/lodash.isstring

Reply:
Done